addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexingDirectContentAccessDecorator.java

```diff
@@ -57,6 +63,12 @@ public Transfer retrieveRaw( final ArtifactStore store, final String path, final
         {
             return transfer;
         }
+        else if ( indexCfg.isAuthoritativeIndex() )
```

Comment:
Does this need to check if the store's authoritativeIndex flag is set before returning null?